pwndbg/ghidra.py

```diff
@@ -10,6 +10,10 @@
 import pwndbg.radare2
 import pwndbg.rizin
 
+r2decompiler = pwndbg.gdblib.config.add_param(
+    "r2decompiler", "radare2", "framework that your ghidra plugin installed (radare2/rizin)"
```

Comment:
Can we add a trigger that will validate the provided value to be one of the expected ones?

See an example here:
https://github.com/pwndbg/pwndbg/blob/16a0f3b57ba281d14a774b8bd019a912d7c64655/pwndbg/glibc.py#L38-L50

Reply:
it's done. 
![image](https://github.com/pwndbg/pwndbg/assets/79578430/cbc4c8ad-5e81-4c93-b9a2-c90feb416e41)